### ISSUE TITLE
Missing Code

### ISSUE THREAD

**Issue by MuhamaadUsman** [KutuDev/ICR#1]
Hi There is missing code in Notebook
![image](https://github.com/KutuDev/ICR/assets/50642248/12852e96-5256-43d9-92f8-85329fdfefac)

**Comment by KutuDev** [comment]
Yes, I am aware. The work has stalled. I am sorry. I need to archive the project ASAP.
